lib/environment/metadata_agent.ex

```diff
@@ -30,4 +33,18 @@ defmodule ApplicationRunner.Environment.MetadataAgent do
       end
     )
   end
+
+  def create_metadata(env_id) do
```

Comment:
Ok I'm going to propose a little refacto, with what I see we can remove this function, replace the metadata created in the socket and remove the adapter from the configuration.
Actually a cron will only be for one application and one send, so instead of making this change that impacts the server/devtool we can :
- Add service_name in the Cron schema
- Pass it to the run_env_cron fun
- In run_env_cron you can create the metadata (call AppSocket.de_create_env_token for the token)

(for the name of the service, you can find it in the session_metadata, you can add a function in MetadataAgent get_function_name it can be useful in the future)

If I haven't made myself clear, tell me and I'll explain in more detail.